.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+Thank you for your contribution to Dynatrace's Configuration as Code tool `monaco`.\
+Before submitting this Pr, please make sure that:
+
+- [ ] Your code builds without any errors or warnings.
```

Comment:
This is ensured with our merge prerequisites - if it is not buildable, merge is impossible